src/include/iscsi-lio-server/dialogs.rb

```diff
@@ -485,7 +485,6 @@ def AddDialog
         VStretch()
       )
 
-      help = CWM.MergeHelps(w)
```

Comment:
Um, I do not know what that `help` actually contains, but maybe we should merge that built help with the `@HELPS["target-add"]` below?

Reply:
the help were removed because wasn't used at all, and instead the @HELPS["target-add"] was given. But as it is SP4 specific and in master we are already using CWM, I have just reverted the changes.